tests/detached_editing/test_serialization.py

```diff
@@ -0,0 +1,139 @@
+import json
+import unittest
+from base64 import b64decode, b64encode
+
+from qgis.core import QgsGeometry
+from qgis.PyQt.QtCore import QDate, QDateTime, QTime
+
+from nextgis_connect.detached_editing.serialization import (
+    deserialize_geometry,
+    deserialize_value,
+    serialize_geometry,
+    serialize_value,
+)
+from tests.ng_connect_testcase import (
+    NgConnectTestCase,
+)
+
+
+class TestSerialization(NgConnectTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.types = [
+            (12345, "12345", 12345),  # integer
```

Comment:
Please create a dataclass for this datasets. Without names it is difficult to understand the meaning without reading the tests code